.github/workflows/workflow.yml

```diff
@@ -207,7 +207,34 @@ jobs:
       - name: Check aea dependenices and imports
         run: |
           tox -e dependencies_check
-
+  plugins_install_check:
+    continue-on-error: False
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python_version: [3.6]
+    timeout-minutes: 10
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Install tox
+        run : |
+          pip install tox
+      - name: Check plugin aea-ledger-cosmos
+        run : |
+          tox -r -e plugins_env -- /bin/sh -c "pip install ./plugins/aea-ledger-cosmos && aea generate-key cosmos && echo aea-ledger-cosmos checked!"
```

Comment:
That won't work. You need to install the local version. Not the one from pypi. (see how its done in tox.ini)

Reply:
it's done in tox.ini as a dependency: `deps = .[all]`